lib/MusicBrainz/Server/Data/WikipediaExtract.pm

```diff
@@ -75,8 +75,8 @@ sub get_extract_by_language
 sub get_available_languages
 {
     my ($self, $links, %opts) = @_;
-    my ($url_pattern, $key, $callback, $language, $ret);
     for my $link (@$links) {
+        my ($url_pattern, $key, $callback, $language, $ret);
         if ($link->isa('MusicBrainz::Server::Entity::URL::Wikidata')) {
             $url_pattern = "https://www.wikidata.org/w/api.php?action=wbgetentities&format=json&props=sitelinks&ids=%s%s";
```

Comment:
The problem is that this code, in a not particularly elegant way, uses the same method for both Wikidata and Wikipedia, and therefore expects two parameters that can be put where `%s` is. If `$language` is `undef`, it is replaced with the empty string, and the result is `...&ids=Q1234`, as desired; however, if `$language="en"` from a previous round, the result will be `...&ids=enQ1234`. Solving it more elegantly would be a bit more of a refactoring, though, affecting other code as well.